packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -6,7 +6,8 @@
 
 ### Accessibility Improvements
 - [a11y] Hide svg from screenreader as they are decorative on homepage [#1980](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1980) 
-
+- [a11y]] Hide log out svg from screenreader as they are decorative [#2000](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2000)
+- [a11y] Ensure heading level matches the heading's visual importance/level [#2000](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2000)
 
```

Comment:
Optional: add a section for adding accessibility text for the menu drawer on mobile

Reply:
ah good catch. Done